keyboards/mokey/mokey12x2/keymaps/via/rules.mk

```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```

Comment:
You need to add `ENCODER_MAP_ENABLE = yes` to `rules.mk`, and then add the [encoder map](https://docs.qmk.fm/#/feature_encoders?id=encoder-map) to `keymap.c`; then adjust the VIA JSON that you should be submitting to [the-via/keyboards](https://github.com/the-via/keyboards/) according to the [VIA docs on encoder support](https://www.caniusevia.com/docs/layouts#rotary-encoders).  Do the same change for the `default` keymap too (encoder map is now considered to be the preferred way to configure encoders).

However, do not set `ENCODER_MAP_ENABLE = yes` at the keyboard level — this option is allowed only at the keymap level.